src/licensedcode/index.py

```diff
@@ -874,6 +885,25 @@ def match(
             # break if deadline has passed
             if time() > deadline:
                 break
+        
+        # refining matches without filtering false positives
+        matches, _discarded = match.refine_matches(
+            matches=matches,
+            idx=self,
+            query=qry,
+            min_score=min_score,
+            filter_false_positive=False,
+            merge=True,
+        )
+
+        original_qspan = Span(0, len(qry.tokens)-1)
```

Comment:
What is `len(qry.tokens) - 1` ? coulf you assign it to a variable so that it has a name and is easier to understand?